source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -27,6 +27,19 @@ The keywords “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL N
 Definitions
 ===========
 
+Connection
+~~~~~~~~~~~~~~
+
+A Connection (when linked) refers to the ``Connection`` type defined
+in this specification. It does not refer to an actual TCP/IP
+connection to an Endpoint. A ``Connection`` will attempt to create and
```

Comment:
Could "TCP/IP connection" be shortened to "TCP connection" and then used everywhere instead of plain lowercased "connection", such as on line 36 immediately below?